src/User.java

```diff
@@ -0,0 +1,111 @@
+import java.util.Scanner;
+
+/**
+ * Created by melmo on 11/30/16.
+ */
+public class User {
+    private String name;
+    private int balance;
+    private Scanner input = new Scanner(System.in);
+
+    public User(int startBalance) {
+        this.balance = startBalance;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public int getBalance() {
+        return this.balance;
+    }
+
+    public void chooseName() throws Exception {
+        System.out.println("What is your name?");
+        String name = this.input.nextLine();
+        if (!name.equals("")) {
+            this.name = name;
+            System.out.format("Hello %s.\n", this.name);
+        } else {
+            throw new Exception("Invalid input!");
```

Comment:
Nitpicky but for user interfaces it is good to provide what happened. You didn't enter any characters or the name must be in the form of x will help your users.